OpenFAST/IEA-15-240-RWT/IEA-15-240-RWT_AeroDyn15.dat

```diff
@@ -1,39 +1,39 @@
-False                    CompAA       - Flag to compute AeroAcoustics calculation [used only when WakeMod = 1 or 2]
-"AeroAcousticsInput.dat" AA_InputFile - AeroAcoustics input file [used only when CompAA=true]
+False                  Echo        - Echo the input to "<rootname>.AD.ech"?  (flag)
+default                DTAero      - Time interval for aerodynamic calculations {or "default"} (s)
+1                      WakeMod     - Type of wake/induction model (switch) {0=none, 1=BEMT, 2=DBEMT, 3=OLAF} [WakeMod cannot be 2 or 3 when linearizing]
+2                      AFAeroMod   - Type of blade airfoil aerodynamics model (switch) {1=steady model, 2=Beddoes-Leishman unsteady model} [AFAeroMod must be 1 when linearizing]
+1                      TwrPotent   - Type tower influence on wind based on potential flow around the tower (switch) {0=none, 1=baseline potential flow, 2=potential flow with Bak correction}
+1                      TwrShadow   - Calculate tower influence on wind based on downstream tower shadow (switch) {0=none, 1=Powles model, 2=Eames model}
+True                   TwrAero     - Calculate tower aerodynamic loads? (flag)
+False                  FrozenWake  - Assume frozen wake during linearization? (flag) [used only when WakeMod=1 and when linearizing]
+False                  CavitCheck  - Perform cavitation check? (flag) [AFAeroMod must be 1 when CavitCheck=true]
+False                  CompAA      - Flag to compute AeroAcoustics calculation [only used when WakeMod=1 or 2]
+AeroAcousticsInput.dat AA_InputFile - AeroAcoustics input file [used only when CompAA=true]
 ======  Environmental Conditions  ===================================================================
  1.225000000000000e+00 AirDens     - Air density (kg/m^3)
- 1.479232653061225e-05 KinVisc     - Kinematic air viscosity (m^2/s)
- 3.350000000000000e+02 SpdSound    - Speed of sound (m/s)
+ 1.477551020408163e-05 KinVisc     - Kinematic air viscosity (m^2/s)
+ 3.400000000000000e+02 SpdSound    - Speed of sound (m/s)
  1.035000000000000e+05 Patm        - Atmospheric pressure (Pa) [used only when CavitCheck=True]
  1.700000000000000e+03 Pvap        - Vapour pressure of fluid (Pa) [used only when CavitCheck=True]
  5.000000000000000e-01 FluidDepth  - Water depth above mid-hub height (m) [used only when CavitCheck=True]
 ======  Blade-Element/Momentum Theory Options  ====================================================== [used only when WakeMod=1]
 2                      SkewMod     - Type of skewed-wake correction model (switch) {1=uncoupled, 2=Pitt/Peters, 3=coupled} [unused when WakeMod=0 or 3]
-"default"            SkewModFactor - Constant used in Pitt/Peters skewed wake model {or "default" is 15/32*pi} (-) [used only when SkewMod=2; unused when WakeMod=0 or 3]
+1.4726215563702154     SkewModFactor - Constant used in Pitt/Peters skewed wake model {or "default" is 15/32*pi} (-) [used only when SkewMod=2; unused when WakeMod=0 or 3]
```

Comment:
Better to leave as "default"?

Reply:
not sure what default means, but I went back to default